frontend/src/pages/teams/new.tsx

```diff
@@ -35,23 +35,25 @@ const NewTeam: NextPage = () => {
 	useEffect(() => {
 		const listMembers: TeamUser[] | undefined = [];
 
-		data?.forEach((user) => {
-			if (user._id === session?.user.id) {
-				listMembers.push({
-					user,
-					role: TeamUserRoles.ADMIN,
-					isNewJoiner: false
-				});
-			}
-		});
+		if (data) {
+			data.forEach((user) => {
+				if (user._id === session?.user.id) {
+					listMembers.push({
+						user,
+						role: TeamUserRoles.ADMIN,
+						isNewJoiner: false
+					});
+				}
+			});
 
-		const usersWithChecked = data?.map((user) => {
-			return { ...user, isChecked: user._id === session?.user.id };
-		});
+			const usersWithChecked = data?.map((user) => {
```

Comment:
"data" exists inside the "if" statement, I would remove the "?" from "data".
Maybe I would check if "data" is an Array stead if it is truthy.  
```suggestion
			if(Array.isArray(data)) {
```